app/core/templates/core/profile/index.html

```diff
@@ -16,3 +16,11 @@
     <hr>
     {% include 'core/profile/questions.html' %}
 {% endblock %}
+
+{% block 'meta' %}
+    <meta property="og:title" content="OSS-Challenge" />
```

Comment:
This is most likely not the title you want..

Reply:
Oops my fault, sorry.